addons/base_import/models/base_import.py

```diff
@@ -857,33 +1088,34 @@ def _import_image_by_url(self, url, session, field, line_number):
             response.raise_for_status()
 
             if response.headers.get('Content-Length') and int(response.headers['Content-Length']) > maxsize:
-                raise ValueError(_("File size exceeds configured maximum (%s bytes)", maxsize))
+                raise ImportValidationException(_("File size exceeds configured maximum (%s bytes)", maxsize), field=field)
 
             content = bytearray()
             for chunk in response.iter_content(DEFAULT_IMAGE_CHUNK_SIZE):
                 content += chunk
                 if len(content) > maxsize:
-                    raise ValueError(_("File size exceeds configured maximum (%s bytes)", maxsize))
+                    raise ImportValidationException(_("File size exceeds configured maximum (%s bytes)", maxsize), field=field)
 
             image = Image.open(io.BytesIO(content))
             w, h = image.size
             if w * h > 42e6:  # Nokia Lumia 1020 photo resolution
-                raise ValueError(
-                    u"Image size excessive, imported images must be smaller "
-                    u"than 42 million pixel")
+                raise ImportValidationException(_("Image size excessive, imported images must be smaller than 42 million pixel"), field=field)
```

Comment:
I just searched "nokia lumia 1020" on Google, talking about "reminiscence of the past" ...